internal/preferences.go

```diff
@@ -68,6 +68,7 @@ func (p *InMemoryPreferences) remove(key string) {
 	delete(p.values, key)
 }
 
+// FireChange causes InMemoryPreferences to activate all its .changeListeners
```

Comment:
The case mis-match here due to it not being exported any more will (now or in the future?) trip the linter...

Reply:
After the latest round of changes, this added comment is the only thing changed in this file. I'll remove it to return the file back to the original state (besides, this particular function is simple and does not require an extra comment).